libs/domain/product/price/src/price.component.ts

```diff
@@ -35,10 +28,44 @@ export class ProductPriceComponent extends ProductMixin(
 ) {
   static styles = ProductPriceStyles;
 
-  protected pricingService = resolve(PricingService);
+  /**
+   * Indicates the sales price. If the sales price is not given by a property,
+   * it will be resolved from the product data.
+   */
+  @signalProperty() sales?: number;
+
+  /**
+   * The component uses the `PriceComponent` to render prices in the active currency.
+   * If a currency is provided, the price currency must match this currency, otherwise the price
+   * is not rendered
+   */
+  @property() currency?: string;
+
+  protected $salesPrice = computed(() => {
+    const { defaultPrice } = this.$product()?.price ?? {};
+    return (
+      (!this.currency || this.currency === defaultPrice?.currency) &&
+      defaultPrice?.value
+    );
+  });
+
+  protected $originalPrice = computed(() => {
+    const { originalPrice } = this.$product()?.price ?? {};
+    return (
+      (!this.currency || this.currency === originalPrice?.currency) &&
+      originalPrice?.value
+    );
+  });
 
-  protected $prices = computed(() => {
-    return this.formatPrices(this.$product()?.price);
+  protected $isDiscounted = computed(() => {
+    return (
+      (this.sales &&
+        typeof this.$salesPrice() === 'number' &&
+        this.sales < Number(this.$salesPrice())) ||
+      (typeof this.$salesPrice() === 'number' &&
+        typeof this.$salesPrice() === 'number' &&
```

Comment:
Why we need to identical conditions?

Reply:
Thanks, that was wrong. I've actually updated the logic and aligned e2e.

Reply:
Also renamed the `SitePriceComponent`, as it had a flaw.